tools/collaboration.md

```diff
@@ -0,0 +1,91 @@
+- Automated meeting recording and transcription, making those available to named meeting participants via their LFX Individual Dashboard and, more broadly, via the Public Meeting Calendar if the meeting is set to `Public`.
+Meeting attendance is taken automatically and attached to the meeting itself. Attendance is reportable to gauge participants' engagement.
+
+### Adding meetings
+
+Instructions for creating a meeting in PCC can be found [here][create meeting instructions]. Specific guidelines for hosted projects and working groups should be as follows.
+
+- Generally, all hosted projects and working group meetings should have `Make Meeting Public` checked and `Restricted` unchecked, as those meetings are public by default. Occasionally, meetings related to security vulnerability review or other sensitive topics may be held; in those cases, `Make Meeting Public` should be unchecked and `Restricted` checked.
+- Associate a `Committee` with the meeting to ensure that those committee members are automatically added. Other non-committee members can join using the meeting link and request that a meeting invite be sent to them.
+
+### Cancelling meetings
+
+Instructions for canceling meetings can be found [here][cancel meeting instructions].
+
+## Calendar
+
+Each hosted project and working group has a Public Calendar that can be directly pointed to or embedded in an existing web page. Instructions for getting the link for the hosted project or working group Public Calendar can be found [here][access to public calendar link instructions]. Note that the LF Staff sets up a subdomain forward if the project has a primary domain setup using the format `https://calendar.DOMAINNAME`.
+
+All public meetings for hosted projects and working groups are posted on the [Community Calendar]. 
+
+Meetings that an individual is directly invited to can be found in the 'Meetings' section of that individual's LFX Individual Dashboard; instructions can be found [here][individual dashboard meetings instructions]
+
+## Best Practices
+
+To help individuals not be inundated with calendar overload, try to follow these guidelines:
+
+- Make sure to make meeting invite changes ( including cancellations or reschedulings ) more than 24 hours in advance, as many calendaring clients wait to update. The best practice is to email the respective committee's mailing list or [email the list of meeting attendees in PCC][email meeting participants PCC] to inform the group of the cancellation.
+- Instruct invitees to manually remove old entries when provisioning a new series of recurring meetings. Occasionally, we will see some calendaring clients not removing these automatically.
+- Ensure the meeting description includes an agenda or a link to the agenda.
+
+[Community Calendar]: https://calendar.openmainframeproject.org
```

Comment:
This link points to the Open Mainframe Project calendar, and should point here instead:
https://calendar.aswf.io

Reply:
Good catch - fixed now